packages/big-design/src/components/MultiSelect/types.ts

```diff
@@ -1,12 +1,12 @@
 import { Placement } from '@popperjs/core';
-import React, { RefObject } from 'react';
+import React, { ComponentPropsWithoutRef, RefObject } from 'react';
 
 import { InputProps } from '../Input';
 import { InputLocalization } from '../Input/Input';
 import { SelectAction, SelectOption } from '../Select';
 import { SelectOptionGroup } from '../Select/types';
 
-interface BaseSelect extends Omit<React.HTMLAttributes<HTMLInputElement>, 'children'> {
+interface BaseSelect extends Omit<ComponentPropsWithoutRef<'input'>, 'children' | 'value'> {
```

Comment:
This was the only type I actually had to change due to a type issue. We are using our own custom `value` prop rather than the intrinsic input element `value` attribute so we needed to omit it from the types.